tools/generate_helper_code_from_value_sets.rb

```diff
@@ -50,31 +50,50 @@
+    "http://hl7.org/fhir/administrative-gender" => "VRDR.CodeSystems.AdministrativeGender",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-bypass-edit-flag-cs" => "VRDR.CodeSystems.BypassEditFlag",
+    "http://terminology.hl7.org/CodeSystem/v3-EducationLevel" => "VRDR.CodeSystems.EducationLevel",
+    "http://terminology.hl7.org/CodeSystem/v2-0360" => "VRDR.CodeSystems.DegreeLicenceAndCertificate",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-pregnancy-status-cs" => "VRDR.CodeSystems.PregnancyStatus",
+    "http://hl7.org/fhir/us/vrdr/CodeSystem/vrdr-missing-value-reason-cs" => "VRDR.CodeSystems.MissingValueReason",
+    "http://unitsofmeasure.org" => "VRDR.CodeSystems.UnitsOfMeasure",
+    "http://terminology.hl7.org/CodeSystem/v2-0136" => "VRDR.CodeSystems.YesNo"
 }
 valuesets = {
-    "ValueSet-PHVS-DecedentEducationLevel-NCHS.json" => "EducationLevel",
-    "ValueSet-PHVS-MannerOfDeath-NCHS.json" => "MannerOfDeath",
-    "ValueSet-PHVS-MaritalStatus-NCHS.json" => "MaritalStatus",
-    "ValueSet-PHVS-PlaceOfInjury-NCHS.json" => "PlaceOfInjury",
-    "ValueSet-PHVS-PlaceOfDeath-NCHS.json" => "PlaceOfDeath",
-    "ValueSet-PHVS-TransportationRelationships-NCHS.json" => "TransportationRoles",
-    "ValueSet-PHVS-PregnancyStatus-NCHS.json" => "PregnancyStatus",
-    "ValueSet-PHVS-MethodsOfDisposition-NCHS.json" => "MethodsOfDisposition",
-    "ValueSet-PHVS-CertifierTypes-NCHS.json" => "CertificationRole"
+    "ValueSet-vrdr-administrative-gender-vs.json" => "AdministrativeGender",
+    "ValueSet-vrdr-certifier-types-vs.json" => "CertifierTypes",
+    "ValueSet-vrdr-contributory-tobacco-use-vs.json" => "ContributoryTobaccoUse",
+    "ValueSet-vrdr-edit-bypass-01-vs.json" => "EditBypass01",
+    "ValueSet-vrdr-edit-bypass-012-vs.json" => "EditBypass012",
+    "ValueSet-vrdr-edit-bypass-01234-vs.json" => "EditBypass01234",
+    "ValueSet-vrdr-edit-bypass-0124-vs.json" => "EditBypass0124",
+    "ValueSet-vrdr-education-level-vs.json" => "EducationLevel",
+    "ValueSet-vrdr-manner-of-death-vs.json" => "MannerOfDeath",
+    "ValueSet-vrdr-marital-status-vs.json" => "MaritalStatus",
+    "ValueSet-vrdr-method-of-disposition-vs.json" => "MethodOfDisposition",
+    "ValueSet-vrdr-not-applicable-vs.json" => "NotApplicable",
+    "ValueSet-vrdr-place-of-death-vs.json" => "PlaceOfDeath",
+    "ValueSet-vrdr-pregnancy-status-vs.json" => "PregnancyStatus",
+    "ValueSet-vrdr-race-missing-value-reason-vs.json" => "RaceMissingValueReason",
+    "ValueSet-vrdr-transportation-incident-role-vs.json" => "TransportationIncidentRole",
+    "ValueSet-vrdr-units-of-age-vs.json" => "UnitsOfAge",
+    "ValueSet-vrdr-yes-no-not-applicable-vs.json" => "YesNoNotApplicable",
+    "ValueSet-vrdr-yes-no-unknown-not-applicable-vs.json" => "YesNoUnknownNotApplicable",
+    "ValueSet-vrdr-yes-no-unknown-vs.json" => "YesNoUnknown"
 }
-basedir = ARGV[0]
+
 outfilename = ARGV[1] + "/ValueSets.cs"
```

Comment:
Are the instructions for autogenerating the value sets documented somewhere?

Reply:
I'm generally updating the scripts elsewhere